types/types.go

```diff
@@ -101,10 +101,10 @@ type Package struct {
 	// 'package x' line.
 	Name string
 
-	// DocComments from doc.go, if any.
+	// The comment right above the package declaration in doc.go, if any.
 	DocComments []string
 
-	// Comments from doc.go, if any.
+	// All comments from doc.go, if any.
```

Comment:
Our distinction of DocComments is a little weird, anyway.  doc.go is a convention, but not a requirement.

For example:  https://godoc.org/k8s.io/gengo/args

There's no doc.go there, and yet it extracts the package comments correctly.